spec/support/shared/shopify_shop.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.shared_context 'shopify_shop' do
+  let(:refund_amount) { 100 }
+  let(:shopify_payment_method) do
+    gateway = Spree::Gateway::ShopifyGateway.new
+    gateway.set_preference(:api_key, ENV.fetch('SHOPIFY_API_KEY'))
+    gateway.set_preference(:password, ENV.fetch('SHOPIFY_PASSWORD'))
+    gateway.set_preference(:shop_name, ENV.fetch('SHOPIFY_SHOP_NAME'))
+    gateway
+  end
+
+  def mock_request(endpoint, extension)
```

Comment:
It's weird though because when looking at the spec, the way you're calling the API seems to reflect how they do it in their testing suite. I don't think there is a param missing:

https://github.com/Shopify/shopify_api/blob/master/test/order_test.rb#L13